.github/workflows/actions.yml

```diff
@@ -0,0 +1,95 @@
+# https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions
+# https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/
+#
+# Needed GitHub Repository Secret:
+#   PYPI_PASSWORD - PyPI API token allowing write, release access to PyPI.
+#     Limit scope to just project.
+#     (See package.python.org example link above.)
```

Comment:
I assume this is copy + paste from somewhere, but actually `PYPY_PASSWORD` is not used, and this GitHub action will never publish to PyPI.